ipaserver/plugins/baseldap.py

```diff
@@ -550,6 +550,7 @@ class LDAPObject(Object):
     uuid_attribute = ''
     attribute_members = {}
     rdn_is_primary_key = False # Do we need RDN change to do a rename?
+    allow_rename = False
```

Comment:
Why introduce a new class attribute when `rdn_is_primary_key` does exactly the same?